web/src/components/layout/footer.tsx

```diff
@@ -58,11 +52,13 @@ export default React.memo(() => {
             <Localized
               id="content-license-text"
               elems={{
-                licenseLink: <a
-                  target="_blank"
-                  rel="noopener noreferrer"
-                  href="https://www.mozilla.org/en-US/foundation/licensing/website-content/"
-                />
+                licenseLink: (
+                  <a
+                    target="_blank"
+                    rel="noopener noreferrer"
+                    href="https://www.mozilla.org/en-US/foundation/licensing/website-content/"
+                  />
+                ),
```

Comment:
Is this just a linter change or was there a functional reason?

Reply:
Bah, yep! Just Prettier doing its thing, thanks for checking.